FSJ_django20_project/FSJ/templates/FSJ/view_application.html

```diff
@@ -63,10 +63,50 @@
         <form action='{{ url }}' method='post'>
             {% csrf_token %}
             {# Save button on application #}
+
             {% if FSJ_user.user_class == "Coordinator" and not archived %}
+                <table class="table table-bordered table-striped table-hover" align="center">
```

Comment:
Should this table exist if no comments have been made? If so, at least an empty record indicating no comments have been made would be preferred to having just the header.